tests/unit/test_io_utils.py

```diff
@@ -44,6 +44,13 @@ def test_assert_path_is_not_empty(self, tmp_path):
                 path=empty_temp_file_path, description="Io Utils"
             ), "Provided a fake file path, function is fail"
```

Comment:
Mind fixing the comment to `"Provided path to an empty file, function expected to fail"`
and the same with the ones above